package.json

```diff
@@ -14,11 +14,18 @@
     "*.js"
   ],
   "scripts": {
-    "postinstall": "git clone https://github.com/api3dao/airnode.git airnode; cd airnode && git checkout 994d895f37ca871f57224ac1603ea58e79ef071e && yarn run bootstrap && yarn build && cd ..; cp -a airnode/node_modules/@api3/airnode-node/. node_modules/@api3/airnode-node; rm -rf airnode",
+    "postinstall": "git clone https://github.com/api3dao/airnode.git airnode; cd airnode && git checkout 994d895f37ca871f57224ac1603ea58e79ef071e && yarn run bootstrap && yarn build && cd ..; cp -a airnode/node_modules/@api3/airnode-node/. node_modules/@api3/airnode-node; cp -a src/test/hardhat.config.ts ./; rm -rf airnode",
```

Comment:
We need the hardhat file in the root folder so I'm copying there after installing, this can be also removed once we don't need the postinstall script.